model-optimizer/mo/ops/broadcast.py

```diff
@@ -69,12 +69,20 @@ def infer(node: Node):
                 node.out_port(0).data.set_value(uni_directional_broadcasting(input_value, target_shape))
             elif node.mode == 'bidirectional':
                 node.out_port(0).data.set_value(bi_directional_broadcasting(input_value, target_shape))
+            elif node.mode == 'explicit':
+                assert node.in_port(2).data.get_value() is not None, 'Non-constant values for axes_mapping are not supported'
+                PermuteInputs().set_input_permutation(node.in_node(2), node, 'output:0', 'axis')
+                axes_mapping = node.in_port(2).data.get_value()
```

Comment:
```suggestion
                axes_mapping = node.in_port(2).data.get_value()
                assert axes_mapping  is not None, 'Broadcast(model="explicit") with dynamic axes_mapping input is not supported. Node: `{}`'.format(node_name)
                PermuteInputs().set_input_permutation(node.in_node(2), node, 'output:0', 'axis')
```

Reply:
Agreed will all comment and corrected asserts and normalizations. Please review